### DISCUSSION TITLE
Uvicorn Workers Issues

### DISCUSSION THREAD

**Discussion by Hemanta1888** [Kludex/uvicorn#1531] [Potential Issue]
**Workers are not working**
----------------------------------
I am trying to use workers to handle multiple requests but **Accept failed on a socket** error always coming. Please refer to the below image

![image](https://user-images.githubusercontent.com/70465689/174844116-02082756-1ac6-4352-85cf-449e595f4d27.png)

Please check it and help me to solve my issue

**Comment by Kludex** [comment]
How can ppl reproduce this?

**Reply by Hemanta1888** [reply]
Don't know why it is producing the error but if you are going to use **uvicorn 0.15.0** version then it is working perfectly but while using the latest version it is throwing an error. Please refer to the below image.
![image](https://user-images.githubusercontent.com/70465689/174870197-570741cf-1f4c-4f7e-9a67-41a2903405e8.png)

The above image code uses **uvicorn 0.15.0** version.

**Reply by euri10** [reply]
what would help @Hemanta1888 is a minimal reproducible example.
please add uvicorn version (you did) but also python version, or print the output of `uvicorn --version`

**Reply by Hemanta1888** [reply]
@euri10  
Currently, I am using **python 3.9.1 version and uvicorn 0.17.6**. If you need more details please feel free to ask me

**Reply by euri10** [reply]
an app example that throws this error
